dev/integration_tests/flutter_gallery/lib/duration_main.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This is in the integration test directory, is there a test that makes use of this? Maybe this should be in the examples directory instead?

Reply:
This is only for a evaluating. Not sth which lands in production :)